docs/configuration.md

```diff
@@ -3,4 +3,5 @@
 ### As a service:
 
 * I will take in environment variables natively to my service
-* I will not use configuration files where I can avoid it over using environment variables
+* Any environment specific configuration should be passed in as an environment variable
```

Comment:
what about a private key / certificate? does that have to go in an as an env var? are there any size limits to kube/docker env vars that might affect this?